src/TextInput.mc

```diff
@@ -79,7 +101,14 @@ class TextInputView extends WatchUi.View {
   }
 
   function enter() {
-    text_ = text_ + alphabet_[limit(cursor_, alphabet_.size())];
+    var selected_char = alphabet_[limit(cursor_, alphabet_.size())];
+    // On some devices long-pressing the select button acts as a shortcut,
+    // adding this secondary way to confirm the input works around this
```

Comment:
Good idea with the checkmark character!

Reply:
Changed it o using » as I could neither find out which devices support unicode and which don't.